developers/academy/zero_to_mvp/103_schema_and_imports/30_import.mdx

```diff
@@ -41,10 +41,6 @@ Optionally, you can manually specify:
 We'll cover these in more detail later on.
 :::
 
-:::warning TODO
```

Comment:
This is just a suggestion: I don't think an example code snippet is necessary here. The tutorial flows well without it and thus I would clean this up by simply removing this TODO field.